sdk/spring/pipeline/spring-cloud-azure-supported-spring.json

```diff
@@ -157,7 +173,7 @@
     "snapshot" : false,
     "supportStatus" : "SUPPORTED",
     "spring-boot-version" : "3.1.12",
-    "spring-cloud-version" : "2022.0.5"
+    "spring-cloud-version" : "NONE_SUPPORTED_SPRING_CLOUD_VERSION"
```

Comment:
Why it's NONE_SUPPORTED_SPRING_CLOUD_VERSION?

Reply:
[Actuator](https://api.spring.io/project_metadata/spring-boot) unsupport versions before 3.2, so I think we need to update the logic of this github actions